app/controllers/ldp/ReadWriteWebControllerGeneric.scala

```diff
@@ -79,7 +79,7 @@ trait ReadWriteWebControllerGeneric extends ReadWriteWebControllerTrait {
   private def allowHeaders(authResult: AuthResult[NamedResource[Rdf]]): List[(String, String)] = {
     val modesAllowed = authResult.authInfo.modesAllowed
     val isLDPC =  authResult.result.isInstanceOf[LocalLDPC[_]]
-    val allow = "Allow" -> {
+    val allow = "Access-Control-Allow-Methods" -> {
```

Comment:
The difference is that the `Allow` header is saying: you can use these methods as you are authenticated now. The `Access-Control-Allow-Methods` field is saying what you can do if a JS script is holding a gun to your head and telling you exactly what to do. There are certain JS scripts that are allowed to take over your brain perhaps with the `Origin` header, but not all. 
  So in the case that the RDF is completely public then the two should be the same. What is really needed is in the WebID profile of the user who is authenticated, for him to specify the JS origins he accepts ( As long as an arbitrary JS can't just edit that file. )